src/app/Helpers/Stats.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Helpers;
+use App\Models\Location;
+use DB;
+use Carbon\Carbon;
+
+class Stats {
+    /**
+     * Get the number of locations in the database.
+     *
+     */
+    public static function countLocations() {
+      return Location::count();
+    }
+
+    /**
+     * Get the number of locations in the database with future vaccine availability.
+     *
+     */
+    public static function countFutureAvailability() {
+      return Location::available()->count();
+    }
+
+    /**
+     * Get a list of the top howMany most active updaters.
+     *
+     * @var howMany Specify the length of the top updaters list.
+     *
+     */
+    public static function topUpdaters(int $howMany) {
+        return DB::select("select count(updated_by_user_id) as update_count, name from availabilities, users where updated_by_user_id = users.id group by updated_by_user_id, name order by update_count desc limit ?", array($howMany));
```

Comment:
It might be good to also check this by time so that it's not just cumulative over all time. We can add that later. There's also a "Laravel way" of doing this rather than raw SQL, though it may be less code-efficient (more lines/characters). I think it would look something like:
`return DB::table('availabilities')
  ->select('name',DB::raw('count(updated_by_user_id) as update_count'))
  ->join('users', 'availabilities.updated_by_user_id','=','users.id')
  ->whereBetween('updated_at',$start,$end)
  ->groupBy('updated_by_user_id')
  ->get();`